src/context/providers/HubbardUContextProvider.js

```diff
@@ -66,6 +81,19 @@ export class HubbardUContextProvider extends mix(JSONSchemaFormDataProvider).wit
         };
     }
 
+    getValenceOrbitals = (element) => {
+        const pseudos = this.methodData?.pseudo || [];
```

Comment:
We should hide the `pseudo` and try to get `atomicOrbitals` directly from `methodDataItem`

Reply:
Maybe we need should use https://github.com/Exabyte-io/prode.js/blob/f88779045d4a2ad657415dad8fdd3a0201acea42/src/include/meta_properties/pseudopotential.js#L4 class and extract `atomicOrbitals` from it

Reply:
`.pseudo` should not be needed, we should have methodDataItems inside `methodData` directly